src/main/scala/com/databricks/spark/sql/perf/mllib/classification/LinearSVC.scala

```diff
@@ -0,0 +1,45 @@
+package com.databricks.spark.sql.perf.mllib.classification
+
+import com.databricks.spark.sql.perf.mllib.OptionImplicits._
+import com.databricks.spark.sql.perf.mllib._
+import com.databricks.spark.sql.perf.mllib.data.DataGenerator
+import org.apache.spark.ml.evaluation.{Evaluator, MulticlassClassificationEvaluator}
```

Comment:
Style: Put databricks imports last, use newline between import groups. You can use Intellij's import organizer to handle this automatically as described here: https://github.com/databricks/scala-style-guide#imports